test/components/views/rooms/wysiwyg_composer/WysiwygComposer-test.tsx

```diff
@@ -144,5 +146,20 @@ describe('WysiwygComposer', () => {
         expect(mockClient.sendMessage).toBeCalledWith('myfakeroom', null, expectedContent);
         expect(screen.getByRole('textbox')).toHaveFocus();
     });
+
+    it('Should focus when receiving an Action.FocusSendMessageComposer action', async () => {
+        // Given we don't have focus
+        customRender(null, false).container;
```

Comment:
I think that the `.container` is not needed here

Reply:
Ah thanks, that was left over from a previous experiment.